src/mpc-hc/PlayerPlaylistBar.cpp

```diff
@@ -402,6 +405,150 @@ bool CPlayerPlaylistBar::ParseBDMVPlayList(CString fn)
     return !m_pl.IsEmpty();
 }
 
+bool CPlayerPlaylistBar::ParseCUESheet(CString fn) {
+    CString str;
+    std::vector<int> idx;
+    int cue_index(0);
+
+    CWebTextFile f(CTextFile::UTF8);
+    if (!f.Open(fn) || !f.ReadString(str)) {
+        return false;
+    }
+
+    f.Seek(0, CFile::SeekPosition::begin);
+    if (f.GetEncoding() == CTextFile::DEFAULT_ENCODING) {
+        f.SetEncoding(CTextFile::ANSI);
+    }
+
+    CString base;
+    bool isurl = fn.Find(_T("://")) > 0;
+    if (isurl) {
+        int p = fn.Find(_T('?'));
+        if (p > 0) {
+            fn = fn.Left(p);
+        }
+        p = fn.ReverseFind(_T('/'));
+        if (p > 0) {
+            base = fn.Left(p + 1);
+        }
+    }
+    else {
+        CPath basefilepath(fn);
+        basefilepath.RemoveFileSpec();
+        basefilepath.AddBackslash();
+        base = basefilepath.m_strPath;
+    }
+
+    bool success = false;
+    CString title(_T(""));
+    CString performer(_T(""));
+    CAtlList<CueTrackMeta> trackl;
+    CAtlList<CPlaylistItem> pl;
+    CueTrackMeta track;
+    int trackID(0);
+
+    while(f.ReadString(str)) {
+        str.Trim();
+        if (cue_index == 0 && str.Left(5) == _T("TITLE")) {
+            title = str.Mid(6).Trim(_T("\""));
+        }
+        else if (cue_index == 0 && str.Left(9) == _T("PERFORMER")) {
+            performer = str.Mid(10).Trim(_T("\""));
+        }
+        else if (str.Left(4) == _T("FILE")) {
+            if (str.Right(4) == _T("WAVE") || str.Right(4) == _T("MP3") || str.Right(4) == _T("AIFF")) { // We just support audio file.

```

Comment:
str.Right(4) == _T("MP3")

Should be 3

Reply:
Are more formats possible? Like FLAC?

Reply:
It should be ```WAVE```.

Reply:
```flac```,```ape``` these files are all treated as ```WAVE```.